src/esssans/beam_center_finder.py

```diff
@@ -158,9 +159,6 @@ def _iofq_in_quadrants(
         'north-west'.
     """
     pi = sc.constants.pi.value
-    phi = data.transform_coords(
-        'phi', graph=graph, keep_intermediate=False, keep_inputs=False
-    ).coords['phi']
     phi_bins = sc.linspace('phi', -pi, pi, 5, unit='rad')
```

Comment:
Take care here when resolving the conflict.

Reply:
I hope I didn't mess up